src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -17,10 +17,15 @@ limitations under the License.
 <configurator>
   <domain name="CHIP"/>
 
-   <struct name="TestListStructOctet">
-     <item name="fabricIndex" type="INT64U"/>
-     <item name="operationalCert" type="OCTET_STRING" length="32"/>
-   </struct>
+  <struct name="TestListStructOctet">
+    <item name="fabricIndex" type="INT64U"/>
+    <item name="operationalCert" type="OCTET_STRING" length="32"/>
+  </struct>
+
+  <struct name="SimpleStruct">
```

Comment:
Better name: TestStructResponse

Reply:
Not quite agree, since this is just a struct for attribute read use, and in code we will use it as`WriteAttribute(..., SimpleStructValue)`.